ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -239,33 +273,30 @@ Once the well-known endpoint for **issuer server** configuration is resolved, th
           "text_color": "#FFFFFF"
         }
       ],
-      "credential_definition": {
```

Comment:
Here, we have some inconsistencies. First, the parameter `credential_definition` is required according to draft 13 as well as to the HAIP profile.

Reply:
Can you elaborate more? We have credential_definition specified here.

Reply:
when format is `vc+sd-jwt` then not using `credential_definition` seems to be correct by draft 13. but `credential_definition` seems to be required by haip for some reason.